src/Autocomplete/src/Form/AutocompleteChoiceTypeExtension.php

```diff
@@ -54,6 +54,10 @@ public function finishView(FormView $view, FormInterface $form, array $options):
 
         $attr = $view->vars['attr'] ?? [];
 
+        if ($form->getConfig()->getType()->getInnerType() instanceof ChoiceType) {
```

Comment:
hmm this does not work if a form type is `AsEntityAutocompleteField`

Reply:
`$form->getConfig()->getType()->getInnerType()->getParent() === BaseEntityAutocompleteType::class` works, but i'll just go with a negation

Reply:
actually i think this is not needed?
TextType can be marked as required too.